save-backend/src/main/kotlin/org/cqfn/save/backend/service/TestService.kt

```diff
@@ -25,7 +27,11 @@ class TestService {
     /**
      * @return Test batches
      */
-    // Fixme: Stub in here
-    fun getTestBatches() = Mono.just(listOf(TestDto("qwe", "www", 0, "id"),
-        TestDto("qwe", "www", 0, "qweqwe")))
+    fun getTestBatches(): Mono<List<TestDto>> {
+        val tests = testRepository.retrieveBatches(offset).map {
+            TestDto(it.expectedFilePath, it.testFilePath, it.testSuiteId, it.id)
+        }
+        offset += 20
```

Comment:
This value is duplicated twice, you should extract it into a constant. I believe, `const val`s are allowed for annotation parameters.